pkg/operator/ceph/controller/spec.go

```diff
@@ -653,8 +655,27 @@ func PrivilegedContext(runAsRoot bool) *v1.SecurityContext {
 	return sec
 }
 
-// LogCollectorContainer runs a cron job to rotate logs
+// LogCollectorContainer rotate logs
 func LogCollectorContainer(daemonID, ns string, c cephv1.ClusterSpec) *v1.Container {
+
+	maxLogSize := "500M"
```

Comment:
Use `resource.NewQuantity` instead of `"500M"`?

Reply:
done